distributed/tests/test_scheduler.py

```diff
@@ -2896,7 +2896,6 @@ async def test_unknown_task_duration_config_2(s, a, b):
 async def test_retire_state_change(c, s, a, b):
     np = pytest.importorskip("numpy")
     y = c.map(lambda x: x**2, range(10))
-    await c.scatter(y)
```

Comment:
No idea why this scatter is here. Am I missing something? Should this be a "replicate" or why would somebody scatter a future??

@jrbourbeau any ideas?